lib/animina/validations/bad_username.ex

```diff
@@ -19,7 +19,7 @@ defmodule Animina.Validations.BadUsername do
     username =
       Ash.Changeset.get_attribute(changeset, opts[:attribute])
 
-    bad_usernames = ["my", "current_user", "profile"]
+    bad_usernames = ["my", "current_user", "profile", "beta"]
 
     if username &&
```

Comment:
While we are at it: Please add a validation, that a username has to have at least 2 characters.